src/index.css

```diff
@@ -183,8 +186,31 @@ h1 {
     background-position: bottom;
     height: 100%;
   }
+
+  .position-set{
+    margin-top: -18px;
+  }
+
+  .story-text {
+    text-align: center;
+    font-size: 18px;
+    font-weight: bold;
+    width: 90%;
+    padding-left: 24px !important;
+    padding-right: 45px;
+   }
+   .storylingo-img{
+    cursor: pointer;
+    height: 15px;
```

Comment:
should not be in pixcel

Reply:
It's was already declared in Pixel. I just moved inline CSS to external css.